crates/jstz_proto/src/context/ticket_table.rs

```diff
@@ -0,0 +1,212 @@
+use crate::error::Result;
+use derive_more::{Display, Error, From};
+use jstz_core::kv::{Entry, Transaction};
+use jstz_crypto::{hash::Blake2b, public_key_hash::PublicKeyHash};
+use tezos_smart_rollup::{
+    host::Runtime,
+    storage::path::{self, OwnedPath, RefPath},
+};
+
+use super::account::Amount;
+
+#[derive(Display, Debug, Error, From)]
+pub enum TicketTableError {
+    InsufficientFunds,
+    AccountNotFound,
+    TicketHashNotFound,
+}
```

Comment:
Off-topic discussion on error handling: I think we should pick a consistent way to handle errors in jstz since currently we use _both_ the `Error` variants or `anyhow`

Reply:
Tracked here: https://linear.app/tezos/issue/JSTZ-40/use-anyhow-errors-in-jstz